src/teleward/captcha.clj

```diff
@@ -10,7 +10,9 @@
 
 (defn add-solution [rands solution cols rows]
   (let [r (rand-int rows)
-        c (rand-int cols)]
+        c (if (== 0 r)
+            (+ 1 (rand-int (- 1 cols)))
+            (rand-int cols))]
```

Comment:
Could you please refactor it a bit? Something like:
```clojure
        c (if (zero? r)
            (inc (rand-int (dec cols)))
            (rand-int cols))]

```

Reply:
sure